conversions/celsius_to_fahrenheit.py

```diff
@@ -0,0 +1,36 @@
+""" Convert temperature from Celsius to Fahrenheit """
+
+
+def celsius_to_fahrenheit(celsius):
+    """
+    Convert a given value from Celsius to Fahrenheit
+
+    >>> celsius_to_fahrenheit(-40)
+    -40.0
+    >>> celsius_to_fahrenheit(-20)
+    -4.0
+    >>> celsius_to_fahrenheit(0)
+    32.0
+    >>> celsius_to_fahrenheit(20)
+    68.0
+    >>> celsius_to_fahrenheit(40)
+    104.0
+    >>> celsius_to_fahrenheit("40")
+    Traceback (most recent call last):
+    ...
+    TypeError: 'str' object cannot be interpreted as integer
+    """
+
+    if type(celsius) == str:
```

Comment:
Please read PEP8's discussion of the use of `isinstance()` instead of comparing types.